base/PerfSettings.php

```diff
@@ -28,8 +29,12 @@ public static function BenchmarkTime(): string {
     return '1M'; // 1 minute
   }
 
+  public static function setBenchmarkConcurrency(int $toSet): void {
+     PerfSettings::$benchmarkConcurrency = $toSet;
+  }
+
   public static function BenchmarkConcurrency(): int {
-    return 200;
+    return PerfSettings::$benchmarkConcurrency;
```

Comment:
Can we actually remove benchmark concurrency settings here, and use the options instead in `base/Siege.php`?  (So basically just make it similar to the way the `serverThreads` option works)
I think it will end up cleaner.